src/main/webapp/js/user-page-loader.js

```diff
@@ -29,13 +29,27 @@ function setPageTitle() {
   document.title = parameterUsername + ' - User Page';
 }
 
+function setAboutMe() {
```

Comment:
This is quite interesting, creating a form on the fly to submit text from elements that are not in a form. I am not very sure how it works though, how come the stuff in the `p` tag are editable?